### ISSUE TITLE
startMove instead of pos=

### ISSUE THREAD

**Issue by levovix0** [treeform/windy#37]
Setting (toplevel) window coordinates (relative to the screen) is not available on Wayland.
Setting window pos often used to let user move window when it is client-side decorated, and this behavior is available on Wayland.
So add function to start interactive window moving and remove or mark as not cross-platform `pos=`

**Comment by treeform** [comment]
I agree we will have to have start/stop window move on platforms that don't have pos=.

Do you know if other platforms support "start/stop window move" or we would have to emulate that?

**Comment by treeform** [comment]
We will have more of these as we support other operating systems like iOS, Android or WASM they will have window properties that are not valid for those systems.

**Comment by levovix0** [comment]
seems like x11 have support, see: https://github.com/qt/qtbase/blob/ac2f4bd9db63cc4a37953b00830ee6214752ea8f/src/plugins/platforms/xcb/qxcbwindow.cpp#L2413
and windows: https://github.com/qt/qtbase/blob/18bc5d652f8845579b27e6a8e87e167a38e7066e/src/plugins/platforms/windows/qwindowswindow.cpp#L2807

**Comment by treeform** [comment]
Feel free to add startMove/stopMove to Wayland windows. Then see if you can do x11? We can eventually add that to win and mac.
